### PR TITLE
Update ome-faq.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to reflect that the new OME supports 150MB as the size limit for encrypted messages.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3561

File: microsoft-365/compliance/ome-faq.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Microsoft 365 users can read and respond from Outlook for Windows and Mac (2013
   
 ## Is there a size limit for messages you can send with OME?
 
-Yes. The maximum message size you can send with OME, including attachments, is 30 MB.
+Yes. The maximum message size you can send with OME(for subscribers using Office 365 Message Encryption with new capabilities), including attachments, is 150 MB.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.md]:
```suggestion
Yes. The maximum message size you can send with OME (for subscribers using Office 365 Message Encryption with new capabilities), including attachments, is 150 MB.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by KCCross** [comment]
This has been to reduced to 25MB per product team.

**Comment by msbemba** [comment]
Thank you @KCCross

**Pull request closed without merging**